### ISSUE TITLE
[BUG]: Document Processor Unavailable

### ISSUE THREAD

**Issue by nuwansamaranayake** [Mintplex-Labs/anything-llm#875]
### How are you running AnythingLLM?

AnythingLLM desktop app

### What happened?

I have installed the desktop app on windows 11. However, I can not upload files as it says "Document Processor Unavailable". I installed on another machine. It works fine. I checked the logs ( whatever I can find) but can not find a clue. Why it works in one machine and not on the other is something that is weird too. Is there any thing we can check to make sure it works

### Are there known steps to reproduce?

Seems like a random issue

**Comment by timothycarambat** [comment]
Related: https://github.com/Mintplex-Labs/anything-llm/issues/866

The application on Windows is likely installed in the wrong user directory and/or permission-related. You can execute the app via powershell in the `C:\\Users\<user>\AppData\Local\Programs\anythingllm-desktop` folder. Running via powershell will print all logs out.

**Comment by nuwansamaranayake** [comment]
Here are the logs. There is a mismatch in prism client and server ??? The user directory is correct. Any idea from the logs ?

Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Datasource "db": SQLite database "anythingllm.db" at "file:C:/Users/nuwan/AppData/Roaming/anythingllm-desktop/storage/anythingllm.db"

Already in sync, no schema change or pending migration was found.

✔ Generated Prisma Client (v5.3.0) to .\node_modules\@prisma\client in 452ms


Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma

✔ Generated Prisma Client (v5.3.0) to .\node_modules\@prisma\client in 433ms

Start using Prisma Client in Node.js (See: https://pris.ly/d/client)
```
import { PrismaClient } from '@prisma/client'
const prisma = new PrismaClient()
```
or start using Prisma Client at the edge (See: https://pris.ly/d/accelerate)
```
import { PrismaClient } from '@prisma/client/edge'
const prisma = new PrismaClient()
```

See other ways of importing Prisma Client: http://pris.ly/d/importing-client

┌─────────────────────────────────────────────────────────────┐
│  Deploying your app to serverless or edge functions?        │
│  Try Prisma Accelerate for connection pooling and caching.  │
│  https://pris.ly/cli/accelerate                             │
└─────────────────────────────────────────────────────────────┘


warn Versions of prisma@5.10.2 and @prisma/client@5.3.0 don't match.
This might lead to unexpected behavior.
Please make sure they have the same version.
[TELEMETRY ENABLED] Anonymous Telemetry enabled. Telemetry helps Mintplex Labs Inc improve AnythingLLM.
prisma:info Starting a sqlite pool with 25 connections.
[TELEMETRY SENT] {
  event: 'server_boot',
  distinctId: '4a083568-d981-41c7-8196-2ee4d571236a',
  properties: { runtime: 'desktop' }
}
[production] AnythingLLM Standalone Backend listening on port 7777
Collector hot directory and tmp storage wiped!
[production] AnythingLLM Standalone Document processor listening on port 21617

**Comment by timothycarambat** [comment]
> [production] AnythingLLM Standalone Document processor listening on port 21617

Seems like it is not able to attach to port 8888 (the expected port). It should be flexible on port attachments. Will investigate further. In the meantime you can find what might be running on 8888 and kill it and reboot the app and that should fix things in the interim.

**Comment by nuwansamaranayake** [comment]
great. yes I found that there is another app using port 8888. The collector should dynamically select a free port for this I guess.

**Comment by timothycarambat** [comment]
Will be patched in the next release of the desktop via #876 

To temporarily workaround for now, find and kill whatever process is occupying port 8888 (typically Jupyter notebook) and AnythingLLM will work

**Comment by timothycarambat** [comment]
fixed ❤️
